environment.yml

```diff
@@ -1,18 +1,20 @@
 name: machine-learning-hats-2021
 channels:
-    - conda-forge
     - pytorch
+    - conda-forge
     - defaults
 dependencies:
     - uproot
     - h5py
-    - keras
-    - tensorflow
+    - tensorflow-mkl>=2
     - scikit-learn
     - scikit-optimize
     - matplotlib
     - pandas
     - pytorch
+    - torchvision
+    - torchaudio
+    - cpuonly
     - ipykernel
```

Comment:
Can you also include `networkx`, which will be used by the new GNN notebook? It's currently being installed with:
```python
pip install networkx
```
so I don't know if there is a conda/mamba package.

Reply:
there is a conda image: https://anaconda.org/anaconda/networkx

Reply:
Yep, I added it when I did my push last night, so once this PR is merged you should be able to remove the pip line from your notebook.